tests/Integration.Tests/DatabaseAccess/CourseDbContextIntegrationTests.cs

```diff
@@ -49,8 +49,8 @@ public void GetLocationFilteredCourses()
 
                 var distance = context2.GetLocationFilteredCourses(50, 0.01, 1000).Single().Distance;
                 Assert.NotNull(distance);
-                Assert.LessOrEqual(715, distance.Value);
-                Assert.GreaterOrEqual(716, distance.Value);
+                Assert.GreaterOrEqual(distance.Value, 713);
+                Assert.LessOrEqual(distance.Value, 714);
```

Comment:
just for my benefit what's the reason behind moving these around?

Reply:
explained in standup - it's because the formula changed so there's a small shift in the numbers